packages/kbn-crypto/tsconfig.json

```diff
@@ -2,6 +2,7 @@
   "extends": "../../tsconfig.bazel.json",
   "compilerOptions": {
     "declaration": true,
+    "declarationMap": true,
```

Comment:
Do we need to define here if it's defined in the `ts_project` rule?

Reply:
Yeah, the two have to be synced unfortunately